### PR TITLE
Detect PHRED variant

### PR DESCRIPTION
**Pull request by wasade**
Few changes necessary for q2 plugin

### CODE REVIEW

File: deblur/workflow.py [hunk 1]
```diff
@@ -32,6 +32,24 @@
 sniff_fastq = skbio.io.io_registry.get_sniffer('fastq')
 
 
+def _get_fastq_variant(input_fp)
```
Comment (mortonjt) [inline deblur/workflow.py]: don't forget the `:` :)

**Comment by wasade** [comment]
...sorry, guilty of using travis to execute the tests...

**Comment by wasade** [comment]
Actually setting up a proper environment. Sorry

**Comment by mortonjt** [comment]
Looks good to me!  @amnona want to merge this?

**Pull request merged**